Form_validation.php

```diff
@@ -0,0 +1,88 @@
+<?php 
+  $valid = array();
+  $errors = array();
+  if ($_POST) {
+
+    $title = $_POST['title'];
+
+    if (!$title) {
+      $errors['title'] = 'Името е задължително поле.';
+      echo $errors['title'];
+      } 
+      elseif (strlen($title) > 150) 
+      {
+        $errors['title'] = 'Името има максимална дължина 150 символа.';
+		echo $errors['title']; 
+      } 
+      else {
+        $valid['title'] = $title;   
+      }
+
+    $teacher = $_POST['teacher'];
+    if (!$teacher) {
+      $errors['teacher'] = 'Името на преподавател е задължително поле.';
+	  echo $errors['teacher'];   
+      } 
+      elseif (strlen($teacher) > 200) 
+      {
+        $errors['teacher'] = 'Името на преподавател има максимална дължина 200 символа.';
+        echo $errors['teacher'];       
+      } 
+      else {
+        $valid['teacher'] = $teacher;   
+      }
+
+      $description = $_POST['description'];
+      if (!$description) {
+        $errors['description'] = 'Описанието на дисциплината е задължително поле.';
+        echo $errors['description'];   
+      } 
+      elseif (strlen($description) < 10) 
```

Comment:
Please extract all constant values into constants/variables and give them meaningful names. This way you will know what is the purpose of the value when reading the code. For example here you would be able to reuse the value of 10 in the error message too here that way.